crates/devolutions-log/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "devolutions-log"
+version = "2024.2.1"
+edition = "2021"
+readme = "README.md"
+license = "MIT/Apache-2.0"
+authors = ["Devolutions Inc. <infos@devolutions.net>"]
+description = "Logging utils library for Devolutions apps"
+publish = false
+
+[dependencies]
+# In-house
+devolutions-gateway-task = { path = "../devolutions-gateway-task" }
```

Comment:
Thought: Maybe we should rename this to `devolutions-task`?

Reply:
This crate was intended for use by Devolutions Gateway only, hence the naming. But we could find a shorter name since it’s not really the case anymore. Even if it’s not for the gateway per se, we could use the `dgw-` prefix for our internal private crates such as this one. (Intended to stay private to the devolutions-gateway repository.)

Reply:
dgw prefix sounds about right, I'll rename both task/log crates to `dgw-task` and `dgw-log`